mods/ra/mod.yaml

```diff
@@ -466,6 +466,72 @@ ModContent:
 					^Content/ra/music/vr2.aud:
 						Offset: 229462230
 						Length: 2920396
+		aftermath: Aftermath Expansion Disc (English)
+			IDFiles:
+				README.TXT: 9902fb74c019df1b76ff5634e68f0371d790b5e0
+				SETUP/INSTALL/PATCH.RTP: 5bce93f834f9322ddaa7233242e5b6c7fea0bf17
+			Install:
+				extract-raw: MAIN.MIX
+					^Content/ra/music/await.aud:
```

Comment:
In any case this is an issue with our music setup rather than our installer, so if something needs to be changed it can be done in another PR.